api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -257,23 +258,27 @@ public void createAllOfUsBillingProject(String projectName, String servicePerime
               projectName, WORKSPACE_DELIMITER));
     }
 
-    FirecloudCreateRawlsBillingProjectFullRequest request =
-        new FirecloudCreateRawlsBillingProjectFullRequest()
-            .billingAccount(configProvider.get().billing.freeTierBillingAccountName())
-            .projectName(projectName)
-            .highSecurityNetwork(true)
-            .enableFlowLogs(true)
-            .privateIpGoogleAccess(true)
```

Comment:
How do these 3 params work in the new API? Are these properties of the perimeter?

Reply:
Those projects are pre-allocated by RBS. There will be different project pools in RBS with different setup. Rawls knows where to find the pool from `servicePerimeter`